src/lib/resolveBaseUrl.ts

```
@@ -0,0 +1,12 @@
+export function resolveBaseUrl(url: string, defaultUrl: string = 'http://localhost') {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)